index.html

```diff
@@ -66,7 +66,7 @@
                                 class="sr-only">(current)</span></a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link font-weight-bold nav-items" href="#about">About</a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html #about">About</a>
```

Comment:
It should be `index.html#about` but still this is not a correct resolution for the issue. We shouldn't reload the page. The issue is with the navbar-collapse look into that.